tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1815,3 +1815,443 @@ def test_work_order_submit_requesterNonce_param_empty(self):
                 is TestStep.SUCCESS.value)
         logger.info('\t\t!!! Test completed !!!\n\n')
 
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_requestersignature_difflength
+    @pytest.mark.listener
+    def test_workordersubmit_requestersignature_difflength(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "work_order_verify_requesterSignature_diff_length.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_submit(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+
+        result_response = self.test_obj.getresult(
+            self.test_obj.build_request_output['request_obj'])
+        assert (
+                verify_test(
+                    result_response, 0,
+                    self.test_obj.build_request_output['pre_test_output'],
+                    self.test_obj.build_request_output['action_obj'])
+                is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_verifyingkey_remove
+    @pytest.mark.listener
+    def test_workordersubmit_verifyingkey_remove(self):
+        request_file = os.path.join(
+            constants.work_order_input_file,
+            "work_order_submit_without_verifyingkey.json")
+
+        err_cd = \
+            self.test_obj.setup_and_build_request_wo_submit(
+                read_json(request_file))
+
+        submit_response = submit_request(
+            self.test_obj.uri_client,
+            self.test_obj.build_request_output['request_obj'],
+            constants.wo_submit_output_json_file_name,
+            read_json(request_file))
+
+        result_response = self.test_obj.getresult(
+            self.test_obj.build_request_output['request_obj'])
+        assert (
+                verify_test(
+                    result_response, 0,
+                    self.test_obj.build_request_output['pre_test_output'],
+                    self.test_obj.build_request_output['action_obj'])
+                is TestStep.SUCCESS.value)
+        logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.work_order_submit
```

Comment:
As i mentioned in my earlier comment, may be this field is not valid and hence may be the test is passing even when null is passed.